tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php

```diff
@@ -224,6 +224,13 @@ public function testRefreshBucketWhenSchemaDoesNotExist(): void
         // test bucket still exists
         $bucket = $this->_client->getBucket($idOfBucket);
         $this->assertNotEmpty($bucket);
+
+        $this->_client->dropBucket($idOfBucket);
+
+        // test bucket is deleted
+        $this->expectException(ClientException::class);
```

Comment:
prosim zabal to do try-catche. Ty `expectExceptionXXX` bych rad prestal pouzivat, protoze to pak dela bordel, kdybys pod to neco jeste pridaval

Reply:
as you wish